README.md

```diff
@@ -8,10 +8,14 @@
 
 ![GitHub Actions](https://github.com/pyMBE-dev/pyMBE/actions/workflows/testsuite.yml/badge.svg)
 [![codecov](https://codecov.io/gh/pyMBE-dev/pyMBE/branch/main/graph/badge.svg)](https://codecov.io/gh/pyMBE-dev/pyMBE)
+[![docs](https://badgen.net/static/docs/up-to-date/yellow/)](pymbe-dev.github.io/pyMBE/pyMBE.html) 
```

Comment:
From a user experience perspective, the color yellow is often associated with low code coverage, low code quality, partially interrupted services and pending changes. In particular, the competing service readthedocs.io uses a very similar badge with yellow when the software documentation is left in an indeterminate state (user manual page on [Status badges](https://docs.readthedocs.io/en/stable/badges.html)). Not to mention the difficulty of reading white text on a yellow background.

I would personally vote for `blue`, which would look like this: [![docs](https://badgen.net/static/docs/up-to-date/blue/)](pymbe-dev.github.io/pyMBE/pyMBE.html)

Reply:
what about having it on `purple` [![docs](https://badgen.net/static/docs/up-to-date/purple/)](pymbe-dev.github.io/pyMBE/pyMBE.html) ?

Just because we are already using blue for the license

Reply:
Fine, let's have it blue for consistency between static banners. Thanks for sharing that post, that made me laugh!